### ISSUE TITLE
[FEATURE] Add Support for Mujoco Playground

### ISSUE THREAD

**Issue by Michael-Beukman** [EdanToledo/Stoix#168]
### Feature
Adding support/wrappers for the [Mujoco playground environments](https://playground.mujoco.org). 
This contains reimplementations of DMC and other robotics tasks, using MJX. But their learning code is based on brax.training, which is hard to modify.

### Proposal
- Add wrappers (e.g. based on [here](https://github.com/google-deepmind/mujoco_playground/issues/81#issue-2920795884)).
- Add dependencies & configs
- Run some tests to reproduce the results in their [paper](https://arxiv.org/abs/2502.08844)

### Testing
A detailed description of how this feature will be tested.
* Mainly, do we get the same performance as they report in their paper using Stoix's PPO & SAC.

### Benchmarking (Optional)
Describe how and why this feature needs to be benchmarked with regards to:
* Do we get expected performance in terms of episodic reward.
* Is the training as fast as we would expect.

### Definition of done
- We can run any stoix system with any of the envs in mujoco playground and (with reasonable hyperparameters) expect it to run fast and perform well.

### Mandatory checklist before making a PR
* [ ] The success criteria laid down in “Definition of done” are met.
* [ ] Code is documented - docstrings for methods and classes, static types for arguments.
* [ ] Code is tested - unit, integration and/or functional tests are added.
* [ ] Documentation is updated - README, CONTRIBUTING, or other documentation.
* [ ] All functional tests are green.
* [ ] Link experiment/benchmarking after implementation (optional).

**Comment by Michael-Beukman** [comment]
Awesome, do let me know if I can help with anything.

It does seem like they use the brax auto reset wrappers for their training code [here](https://github.com/google-deepmind/mujoco_playground/blob/main/mujoco_playground/_src/wrapper.py#L87)


The one other thing is that these DMC-based envs don't terminate when the robot is unhealthy, only when the episode truncation limit is reached.

**Comment by EdanToledo** [comment]
#177 will add mujoco playground support. We can then start evaluating if baselines can be reproduced.

**Comment by EdanToledo** [comment]
So mujoco playground support has been added, i'll make another issue to reproduce their baselines.
